src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/MissingSwitchDefaultCheck.xml

```diff
@@ -17,7 +17,9 @@
  introduction of new types in an enumeration type. Note that
  the compiler requires switch expressions to be exhaustive,
  so this check does not enforce default branches on
- such expressions.
+ such expressions. Also, switch statements that use pattern or null
```

Comment:
@romani Am I missing something that this file looks manually changed and isn't using the description in the javadoc and xdoc?

Reply:
@nmancus1 Are you manually modifying this file?

Reply:
>  Are you manually modifying this file?

No.

Reply:
Such files are updated by test that generate meta data